omp/base/kernel_launch_reduction.hpp

```diff
@@ -290,7 +298,7 @@ void run_kernel_col_reduction_sized_impl(
     syn::value_list<int, remainder_cols>,
     std::shared_ptr<const OmpExecutor> exec, KernelFunction fn, ReductionOp op,
     FinalizeOp finalize, ValueType identity, ValueType* result, dim<2> size,
-    MappedKernelArgs... args)
+    Array<char>& tmp, MappedKernelArgs... args)
```

Comment:
```suggestion
    Array<char>& tmp, MappedKernelArgs&&... args)
```

Reply:
this is deliberate, as MappedKernelArgs need to be value types, not references